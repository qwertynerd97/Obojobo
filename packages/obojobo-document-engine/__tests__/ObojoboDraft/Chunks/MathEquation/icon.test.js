import React from 'react'
import renderer from 'react-test-renderer'

import Icon from 'ObojoboDraft/Chunks/MathEquation/icon'

describe('MathEquation Icon', () => {
	test('Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
