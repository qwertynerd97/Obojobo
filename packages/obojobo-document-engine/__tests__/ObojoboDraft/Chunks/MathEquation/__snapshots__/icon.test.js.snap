// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MathEquation Icon Icon 1`] = `
<svg
  data-name="Layer 1"
  id="Layer_1"
  viewBox="0 0 20.29 14.79"
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <style>
      .cls-1{opacity:0.55;font-size:13px;font-family:Times New Roman;font-style:italic;}
    </style>
  </defs>
  <title>
    equation
  </title>
  <text
    className="cls-1"
    transform="translate(2.25 10.8)"
  >
    f(x)
  </text>
</svg>
`;
